Hamburg/Stabi/SHOr271a.xml

```diff
@@ -1,316 +1,342 @@
+                    <width>145</width>
+                    <depth>40</depth>
+                  </dimensions>
+                  <note>The boards are 6 mm thick and cut to the size of the bookblock.</note>
+                </extent>
 
-                              <collation>
-                                <signatures>
-                                  Quire marks on the upper left corner of the first leaf of each quire (fols. <locus target="#1r #10r #18r #26r #34r"/>).
-                                </signatures>
-                                <list>
-                                  <item xml:id="q1">
-                                    <dim unit="leaf">9</dim>
-                                    <locus from="1" to="9"/>
-                                    1, stub after 9
-                                  </item>
-                                  <item xml:id="q2">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="10" to="17"/>
-                                    6, stub after 2
-                                    1, stub after 5
-                                  </item>
-                                  <item xml:id="q3">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="18" to="25"/>
-                                  </item>
-                                  <item xml:id="q4">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="26" to="33"/>
-                                    6, stub after 3
-                                    3, stub after 6
-                                  </item>
-                                  <item xml:id="q5">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="34" to="41"/>
-                                  </item>
-                                  <item xml:id="q6">
-                                    <dim unit="leaf">6</dim>
-                                    <locus from="42" to="47"/>
-                                  </item>
-                                  <item xml:id="q7">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="48" to="55"/>
-                                  </item>
-                                  <item xml:id="q8">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="56" to="63"/>
-                                  </item>
-                                  <item xml:id="q9">
-                                    <dim unit="leaf">8</dim>
-                                    <locus from="64" to="71"/>
-                                  </item>
+                <collation>
+                  <signatures> Quire marks on the upper left corner of the first leaf of each quire
```

Comment:
If you want to and have the information available, it is possible to encode the quire numbers also within the individual quires. I think it looks good in the visualization :)

Reply:
It means to change Eliana´s work, but I think it is ok to do so.